### PR TITLE
fix: standardize logs & errors

### PR DESCRIPTION
**Pull request by snitin315**
Fixes https://github.com/razorpay/blade/issues/580

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   },
   "scripts": {
     "tsc:blade": "yarn workspace @razorpay/blade typecheck",
-    "lint": "eslint --ext js,jsx,ts,tsx packages && stylelint '**/*.{js,jsx,ts,tsx}'",
+    "lint": "eslint --ext js,jsx,ts,tsx packages/blade --fix && stylelint '**/*.{js,jsx,ts,tsx}'",
```
Comment (saurabhdaware) [inline package.json:21]: intended? maybe create a new script with `lint:blade` name?

File: packages/blade/src/components/Amount/Amount.tsx [hunk 2]
```diff
@@ -232,11 +233,18 @@ const _Amount = ({
...
+      });
     }
     // @ts-expect-error neutral intent should throw error
     if (intent === 'neutral') {
-      throw new Error('[Blade Amount]: `neutral` intent is not supported.');
+      throwBladeError({
+        message: '`neutral` intent is not supported.',
+        moduleName: 'Amount',
+      });
+      throw new Error('[Blade Amount]: ');
```
Comment (saurabhdaware) [inline packages/blade/src/components/Amount/Amount.tsx:247]: miss?

File: package.json (hunk 1 above)
Comment (snitin315) [reply package.json:21]: No, not intended.
```suggestion
    "lint": "eslint --ext js,jsx,ts,tsx packages && stylelint '**/*.{js,jsx,ts,tsx}'",
```

File: packages/blade/src/components/Amount/Amount.tsx (hunk 2 above)
Comment (snitin315) [reply packages/blade/src/components/Amount/Amount.tsx:247]: Yeah, good catch!
```suggestion
```

**Review by saurabhdaware — APPROVED** [review]

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Comment by chaitanyadeorukhkar** [comment]
Add changeset. Lets merge 🚀

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Pull request merged**